src/NUnitFramework/framework/Internal/TestParameters.cs

```diff
@@ -141,7 +141,7 @@ public void ApplyToTest(TestMethod test, TestNameGenerator defaultTestNameGenera
             }
             else
             {
-                test.Name = TestName.Contains("{")
+                test.Name = TestName.IndexOf('{') != -1
```

Comment:
I'm not in love with using a less clear code structure for hypothetical efficiency. If there were a `String.Contains(char)` method, I'd use that. It's quite possible that the internal implementation of `Contains` deals with the special case of a one-character string, but even if it doesn't, we don't really know that we need efficiency here.

Reply:
Fair. I actually agree with you rather strongly so there's that. 😆